pkg/container/container.go

```diff
@@ -276,3 +290,42 @@ func (c *container) SelinuxLabel(sboxLabel string) ([]string, error) {
 	}
 	return labelOptions, nil
 }
+
+// InitSpecGen initializes a new config spec generator.
+func (c *container) InitSpecGen() error {
+	specGen, err := generate.New("linux")
+	if err != nil {
+		return errors.Wrap(err, "creating new linux spec")
+	}
+	c.specGen = &specGen
+	return nil
+}
+
+// SpecGen returns the internal config spec generator.
+func (c *container) SpecGen() *generate.Generator {
+	return c.specGen
+}
+
+// ApplyAppArmor takes the AppArmor config and the profile name as string and
+// applies it to the internal config spec generator.
+func (c *container) ApplyAppArmor(appArmorConfig *apparmor.Config, profileName string) error {
```

Comment:
I've never seen it capatalized as AppArmor. Is that idiomatic? I think I prefer Apparmor

Reply:
Indeed it is `AppArmor` (see the never lying [Wikipedia article](https://en.wikipedia.org/wiki/AppArmor). I also changed that in libraries like containers/common recently (https://github.com/containers/common/pull/260/files)